scss/components/_responsive-embed.scss

```diff
@@ -29,7 +28,10 @@ $responsive-embed-ratio-widescreen: 16 by 9 !default;
 
 /// Creates a responsive embed container.
 /// @param {List} $ratio [$responsive-embed-ratio] - Ratio to use for the container, formatted as `x by y`.
-@mixin responsive-embed($ratio: $responsive-embed-ratio) {
+@mixin responsive-embed($ratio: null) {
```

Comment:
No, I talked about the whole mixin. This is a mixin that set the ratio of a any box, not only `.responsive-embed` and `.flex-video`.